core/src/main/java/org/infinispan/commands/read/EntrySetCommand.java

```diff
@@ -77,7 +81,7 @@ public String toString() {
       @Override
       public CloseableIterator<CacheEntry<K, V>> iterator() {
          Iterator<CacheEntry<K, V>> iterator = new DataContainerRemoveIterator<>(cache);
-         return new EntryWrapperIterator<>(cache, iterator);
+         return Closeables.iterator(iterator);
```

Comment:
Sorry there are a bunch of issues with the `EntryWrapper` for `setValue` when using an iterator or forEach on a stream. I will add this back in. I need to log a JIRA to clean this up though as there are spots missing in different areas. I don't think people were expected to call `setValue` as much :) And usages were just bolted on.